node/process.ts

```diff
@@ -254,7 +259,25 @@ export function kill(pid: number, sig: Deno.Signal | number = "SIGTERM") {
   return true;
 }
 
-class Process extends EventEmitter {
+type NotImplementedListener = EventListenerT;
+type SignalListener = (signal?: number) => void;
+type ProcessListenerMap =
+  & {
+    SIGBREAK: SignalListener;
+    // internally used
+    warning: (warning: Warning) => void;
+    exit: (exitCode?: string | number) => void; // TODO: not sure about exitCode typings
```

Comment:
According to the Node docs, this should be an integer if present I think. When would we want a string here?